Document trusted proxies for IP detection in YOURLS (#461)
YOURLS/website: docs/guide/advanced/proxy.md

```

This setting supports a comma separated list of hosts, using `*` as a wildcard if needed.

## Trusted proxies for IP detection

> When YOURLS runs behind a reverse proxy, load balancer, or CDN (such as Cloudflare, nginx, or AWS ALB), the visitor's real IP address is passed along in the `X-Forwarded-For` HTTP header. By default, YOURLS only trusts `REMOTE_ADDR` and ignores forwarded headers, to prevent IP spoofing.

Reviewer: ```suggestion
When YOURLS runs behind a reverse proxy, a load balancer, or a CDN, the visitor's real IP address is passed along in the `X-Forwarded-For` HTTP header.
By default, YOURLS only trusts `REMOTE_ADDR` and ignores forwarded headers, to prevent IP spoofing.
```

> If you need YOURLS to read the real client IP from forwarded headers, you must explicitly declare which proxy IPs are trusted, using the `get_ip_trusted_proxies` filter in a plugin.

Reviewer: ```suggestion
If YOURLS needs to read the real client IP from forwarded headers, the trusted proxy IPs must be declared, using the `get_ip_trusted_proxies` filter in a plugin.
```
